yarn.lock

```diff
@@ -3452,11 +3452,6 @@ autoprefixer@^9.6.1:
     postcss "^7.0.32"
     postcss-value-parser "^4.1.0"
 
-await-handler@^1.1.2:
-  version "1.1.2"
-  resolved "https://registry.yarnpkg.com/await-handler/-/await-handler-1.1.2.tgz#329c19e9382599898bc12c80c47efdcdaa1b4992"
-  integrity sha512-dihteGhwbJpT89kVbacWiyKeAZr+En0YGK6pAKQJLR0En9ZxSH2H4TTvfG4bBjzFq9gDAma4y9BrpDns6j5UiQ==
-
```

Comment:
...`await-handler` dependency is removed in commit f8f631bc8a80. And that's the only contents of this commit. Can commit 475885465bb be changed so that `await-handler` dependency is not added (apparently it's not needed)? And commit f8f631bc8a80 can thus be removed.

Reply:
Later on, in next PR, await-hander is needed and will be added. I removed it from package.json for this PR, but forgot to run "yarn" to remove it from yarn.lock. But then tests fail, because yarn.lock is not in sync and "yarn install" in test container is trying to create new yarn.lock without write privileges. So I had to update yarn.lock.

Reply:
f8f6 is now removed. await-handler was removed in a new commit that updated yarn.lock anyway and await-handler in not seen anymore as a change.

Reply:
`await-handler` is still added to, and then removed from `yarn.lock`, in this PR. But `await-handler` is not in `package.json` in any commit in this PR. I don't understand why it has to be in `yarn.lock` for a few commits. What brings it there?

Reply:
yarn.lock is an autogenerated file and should not be edited by hand. await-handler was previously in package.json, so it is also here. This commit was edited afterwards. But lock file do not change unless yarn in also run. Running yarn separately just for this commit is pointless.

Reply:
All yarn.lock updates are now reset from commits and created a new commit just for new yarn.lock.

Reply:
> yarn.lock is an autogenerated file and should not be edited by hand.

Yes, that's clear.

> await-handler was previously in package.json, so it is also here.

What do you mean by "was previously"? Running `git log -S await-handler` while this PR's branch is checked out, gives no results, so it's not found anywhere in the commit history.

> This commit was edited afterwards. But lock file do not change unless yarn in also run. Running yarn separately just for this commit is pointless.

No. Yarn (or whatever package management tool is used) **must** be run every time the dependency list file (e.g. `package.json`) is changed. And the lock file **must** be included in the same commit. If yarn was not run for a commit where `package.json` was changed, you use interactive rebase to go back to that commit that modified `package.json`, run yarn and rewrite the commit adding the changed lock file.

> All yarn.lock updates are now reset from commits and created a new commit just for new yarn.lock.

That's wrong. `Package.json` and related `yarn.lock` changes need to be in the same commit. You can rewrite history as long as this is a PR we are talking about.

Reply:
await-handler cannot be found in commit history, because that change in package.json is not in any commit anymore. That caused this discussion as it was in one commit and removed in another. 

yarn.lock in now back in each commit that alters package.json.